lib/linter-views.coffee

```diff
@@ -131,75 +132,47 @@ class LinterViews
       )
       throw null
 
-  updateCurrentLine: (line) ->
-    if @currentLine isnt line
-      @currentLine = line
-      @updateLineMessages()
-      @renderPanel()
-
-
-  updateLineMessages: ->
-    activeEditor = atom.workspace.getActiveTextEditor()
-    @linter.eachEditorLinter (editorLinter) =>
-      return unless editorLinter.editor is activeEditor
-
-      @lineMessages = []
+  updateLineMessages: (shouldRender = false) ->
+    return unless @tabs['Line'].visibility
+    @lineMessages.clear()
+    activeTextEditor = atom.workspace.getActiveTextEditor()
```

Comment:
That was just one single commit that I already did. :sweat_smile: 
https://github.com/AtomLinter/Linter/commit/cfc6cd8630f24bc51aaee3176214c5ed3981f65b